db/migrate/20220718131723_create_wheel_configurations.rb

```diff
@@ -0,0 +1,11 @@
+class CreateWheelConfigurations < ActiveRecord::Migration[7.0]
+  def change
+    create_table :wheel_configurations do |t|
+      t.references :wheel
+      t.string :color_1
+      t.string :color_2
```

Comment:
Why not call this something more explicit like `main_color` and `secondary_color`? Just a suggestion comment, don't need to update if you all agree with these namings.